src/rebar_config.erl

```diff
@@ -107,34 +125,53 @@ write_lock_file(LockFile, Locks) ->
                                            format_attrs(Attrs)]))
     end.
 
-%% Attributes have a special formatting to ensure there's only one per
+%% @private Attributes have a special formatting to ensure there's only one per
 %% line in terms of pkg_hash, so we disturb source diffing as little
 %% as possible.
```

Comment:
this doesn't make much sense as a doc string, but it's private anyways so it prob doesn't matter